napari/_app_model/_app.py

```diff
@@ -1,13 +1,15 @@
 from __future__ import annotations
 
 from functools import lru_cache
+from warnings import warn
 
 from app_model import Application
 
 from napari._app_model.actions._layerlist_context_actions import (
     LAYERLIST_CONTEXT_ACTIONS,
     LAYERLIST_CONTEXT_SUBMENUS,
 )
+from napari.utils.translations import trans
 
 APP_NAME = 'napari'
 
```

Comment:
Lets change the `NapariApplication` method `get_app` to `get_app_model`, and update the `mock_app` as well. Thanks!